src/GitHub.Exports/Models/IBranch.cs

```diff
@@ -1,7 +1,15 @@
-﻿namespace GitHub.Models
+﻿using GitHub.Collections;
+using System;
+
+namespace GitHub.Models
 {
-    public interface IBranch
+    public interface IBranch : ICopyable<IBranch>,
+        IEquatable<IBranch>, IComparable<IBranch>
     {
+        string Id { get; }
         string Name { get; }
+        ISimpleRepositoryModel Repository { get; set; }
+        bool IsTracking { get; set; }
+        string DisplayName { get; set; }
```

Comment:
Remnants of prototyping. `DisplayName` is the only one that's currently modified externally based on what the viewmodel wants the branch to look like in lists.